### PR TITLE
Update default-methods.md

### PR DESCRIPTION
**Pull request by freeaion**
This commit resolves two issues.

a) typo: Equal -> Equals
b) update details to comply w/ example source code

### CODE REVIEW

File: src/traits/default-methods.md [hunk 1]
```diff
@@ -57,4 +57,4 @@ fn main() {
     ```
   * With the blanket implementation, you no longer need `NotEqual` as a super trait for `Equal`.
```
[the remark this answers is not in the document]
Comment (edliaw) [reply src/traits/default-methods.md:58]: Also update the text here.

[the remark this answers is not in the document]
Comment (edliaw) [reply src/traits/default-methods.md:58]: Also update the text here.

**Review by edliaw — APPROVED** [review]

**Comment by mgeisler** [comment]
> @freeaion, could you sign the CLA so we can merge this?

My bad, the problem is actually the commit which uses @edliaw's email address. Let me fix that manually by squashing the commit.

**Comment by edliaw** [comment]
> > @freeaion, could you sign the CLA so we can merge this?
> 
> My bad, the problem is actually the commit which uses @edliaw's email address. Let me fix that manually by squashing the commit.

I think you need to add your review before this can be merged @mgeisler

**Review by mgeisler — APPROVED** [review]

**Comment by mgeisler** [comment]
> I think you need to add your review before this can be merged @mgeisler

Thanks, I clicked all the buttons now and it's merged :smile:

**Pull request merged**